apps/mull-api/src/app/app.module.ts

```diff
@@ -39,7 +38,12 @@ import { EntitiesModule } from './entities';
        * Access the graphql playground at https://localhost:port/graphql
        */
       playground: !environment.production,
+      uploads: {
+        maxFileSize: 10000000000000,
+      },
     }),
+    UserModule,
+    MediaModule,
```

Comment:
You can move these to the top of the imports array, where UserModule was originally.